model/enduser/registry.yaml

```
@@ -0,0 +1,18 @@
+groups:
+  - id: registry.enduser
+    type: attribute_group
+    display_name: End User Attributes
+    brief: >
+      Describes the end user.
+    attributes:
+      - id: enduser.id
+        type: string
+        brief: "Unique identifier of an authenticated user in the system."
+        examples: [ 'username' ]
```

please add a note that it usually contains PII. E.g. like this
```suggestion
        note: |
          > [!Warning]
          > This field contains sensitive (PII) information.
        examples: [ 'username' ]
```